main.go

```diff
@@ -183,7 +183,12 @@ func visit(url *url.URL) {
 		log.Fatalf("unable to create request: %v", err)
 	}
 	for _, h := range httpHeaders {
-		req.Header.Add(headerKeyValue(h))
+		k, v := headerKeyValue(h)
+		if strings.EqualFold(k, "host") {
```

Comment:
Sorry, I take back this statement. I was unaware that `strings.EqualFold()` also included changes in the capitalization of letters.

:+1: